package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "big.js": "^5.2.2",
```

Comment:
I guess you missed this; You need to add `prop-types` and kindly delete all unnecessary files 👍 

![image](https://user-images.githubusercontent.com/31897434/81027149-7ea03300-8e74-11ea-8def-4978f01ee2a2.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/